### ISSUE TITLE
Please add MySQL support

### ISSUE THREAD

**Issue by ghost** [EvolSoft/MassiveEconomy#3]
None of the economy plugins available support MySQL which is troubling seeing as it is 2016. Most of us run dozens of servers and need a centralized economy system.

**Comment by DegenApeDev** [comment]
Yes please finish this awesome economy plugin MySQL support would be so nice!
